test/integration/user_checks_out_from_cart_test.rb

```diff
@@ -16,6 +16,12 @@ class UserChecksOutFromCartTest < ActionDispatch::IntegrationTest
       click_button "Login"
     end
 
+    StripeMock.start
+    customer = Stripe::Customer.create({
+      email: user.email_address,
+      card: '4242424242424242'
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.